src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Also seems to have duplicate code

Reply:
Can we factor out common code of the components? https://reactjs.org/docs/composition-vs-inheritance.html